csss-site/src/events/mountain_madness/templates/mountain_madness/2021/mountain_madness2021_submissions.html

```diff
@@ -0,0 +1,215 @@
+{% extends "csss/header.html" %}
+{% load staticfiles %}
+{% block content %}
+{{ block.super }}
+	<link rel="stylesheet" href="{% static 'mountain_madness_static/2021/csss_custom.css' %}">
+	<section id='above-fold'>
+		<header>
+			<h1 class="title is-size-2">Submissions from the Computing Science Student Society Mountain Madness Hackathon 2021</h1>
```

Comment:
I really don't think this needs to be such a mouthful of a title.

also, If you are gonna have `submissions` here, I don't really see a point to also have it at line 22 at `<h2 class="title is-size-2">Submissions</h2>`

Reply:
can you rename this to `CSSS Mountain Madness Hackathon 2021 Submissions`?